addons/payment/models/payment_token.py

```diff
@@ -108,3 +110,24 @@ def get_linked_records_info(self):
         """
         self.ensure_one()
         return []
+
+    def _get_token_display_name(self, *args, length=8, **kwargs):
+        """ Return a token name of the desired length and format. The default format is •••• 1111.
+        For custom display names of tokens for an acquirer, this method must be overridden.
+
+        Note: self.ensure_one()
+        Note: assert length <= 34
+
+        :param list args: Needed by qWeb when calling this method
+        :param int length: The desired final length of the token name (8 for a bank card)
```

Comment:
This is no longer true for bank cards because the previous function's docstring stated that the length of typical credit card numbers is 16, not 8.
```suggestion
        :param int length: The desired final length of the token name
```